### DISCUSSION TITLE
Running tests with intsize64=on build

### DISCUSSION THREAD

**Discussion by gmargari** [scivision/mumps-superbuild#99] [General]
Hi there guys, and thank you for this immensely helpful project!
 
I compiled MUMPS 5.6.1 on Linux using MKL 2023.1, enabling also complex libs.
When intsize64=off, all 4 test complete successfully (test_mumps_{config,simple32,simple64,complex128}).
When intsize64=on, the test_mumps_complex128 test fails.

Is this something to be expected?
If I understand correctly, the tests are disabled in you github actions so I am not sure  what should be the expected outcome.

Note: tests pass successfully with MUMPS 5.7.3.

Thanks again,
Giorgos

**Comment by scivision** [comment]
Yes you need to have recompiled MPI itself from scratch for intsize64 to work. LAPACK, Scalapack, MPI, etc. must be compiled for Int64.
